docs/application_packages.md

```diff
@@ -263,6 +263,17 @@ setHost. The method is called using JAVA reflection and the property
 value is passed as an argument. In the above example the method setHost
 will be called on the “redis” operator with “127.0.0.1” as the argument.
 
+Properties that are collection types can also be configured, based on the beanutils
+syntax. For example, the connection properties of the JDBC store can be accessed
+like this:
+
+```
```

Comment:
mkdocs seems to render fenced code blocks better if the language is specified (xml in this instance) after the 3 backquotes. Without it we see some odd formatting with some words emboldened, others in a different color, etc. Maybe the 'xml' could also be added to the other blocks as well as appropriate.